examples/log_parsing/inference.py

```diff
@@ -58,28 +57,27 @@ class TritonInferenceLogParsing(TritonInferenceWorker):
         Determines whether a logits calculation is needed for the value returned by the Triton inference response.
     """
 
-    def build_output_message(self, x: MultiInferenceMessage) -> MultiResponseMessage:
-        seq_ids = cp.zeros((x.count, 3), dtype=cp.uint32)
-        seq_ids[:, 0] = cp.arange(x.mess_offset, x.mess_offset + x.count, dtype=cp.uint32)
-        seq_ids[:, 2] = x.get_tensor('seq_ids')[:, 2]
+    def build_output_message(self, msg: ControlMessage) -> ControlMessage:
+        seq_ids = cp.zeros((msg.tensors().count, 3), dtype=cp.uint32)
+        seq_ids[:, 0] = cp.arange(0, msg.tensors().count, dtype=cp.uint32)
+        seq_ids[:, 2] = msg.tensors().get_tensor('seq_ids')[:, 2]
 
-        memory = TensorMemory(
-            count=x.count,
+        memory = _messages.TensorMemory(
+            count=msg.tensors().count,
             tensors={
```

Comment:
We should just use the Python impl of `TensorMemory` if `ControlMessage` doesn't support the the Python implementation of `TensorMemory` we should just update it to support it, mimicing what we did with the `payload_from_python_meta` method.